assets/temp2

```diff
@@ -0,0 +1,301 @@
+<!-- <div class="content-row" id="content-row">
```

Comment:
this file should be `.html` right?

And why do I see a lot of `hard-corded` link, I thought it will be generated by the js file